step.yml

```diff
@@ -145,6 +145,8 @@ inputs:
     opts:
       title: "Untrack blocking versions?"
       description: |-
+        Deprecated, will be ignored.
```

Comment:
We will release a new version with major version # change. Do not keep non-functional inputs as it makes no sense.